mwaz/yummy-recipies
# Challenge two pull request

# What does this PR do
The pull request enables one to create recipe categories, to update recipe categories and to delete recipe categories. Each of the recipe categories has recipes which can also be deleted and updated. The PR has two classes Recipe and Users, the Recipe class holds methods for objects that inherit the Recipe class capabilities including creating recipe categories, updating the recipe categories and even deleting the recipe categories. The Users class, on the other hand, contains methods that are used to create objects for the users such as the registration of a new member and the member login functionalities 

# Descriprion of task to be completed 
The task to be completed is a Test-driven development approach that uses OOP to create objects which can then be utilized by methods that call the classes

# How should this be manually tested
One should clone the repo, install a virtual environment and install the files required to run the app in requirements.txt

# Any background context you want to provide?
The APP utilizes CRUD, basically, it does Create, Update, Delete and Read data from objects that are stored in data structures either lists, dictionaries or even tuples.

# What are the relevant pivotal tracker stories?
Relevant pivotal tracker stories can be found here -> https://www.pivotaltracker.com/n/projects/2119558
